<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgWidget</class>
 <widget class="QWidget" name="MsgWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>68</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,0">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID/Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MsgIDLineEdit" name="msgIDWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="2,0">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Code/Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MsgCodeLineEdit" name="msgCodeWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="6,0">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MsgDataLineEdit" name="msgDataWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MsgCodeLineEdit</class>
   <extends>QWidget</extends>
   <header location="global">MsgCodeLineEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MsgIDLineEdit</class>
   <extends>QWidget</extends>
   <header location="global">MsgIDLineEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MsgDataLineEdit</class>
   <extends>QWidget</extends>
   <header location="global">MsgDataLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
